contrib/test_jenkins.sh

```diff
@@ -414,15 +414,27 @@ run_uct_hello() {
 run_ucx_perftest_mpi() {
 	ucx_inst_ptest=$ucx_inst/share/ucx/perftest
 
+	run_cuda_test=0
+	#check if cuda tests can run
+	if (lsmod | grep -q "nv_peer_mem") && (lsmod | grep -q "gdrdrv")
+	then
+		run_cuda_test=1
+	fi
+
 	# hack for perftest, no way to override params used in batch
 	# todo: fix in perftest
 	sed -s 's,-n [0-9]*,-n 1000,g' $ucx_inst_ptest/msg_pow2 | sort -R > $ucx_inst_ptest/msg_pow2_short
-	cat $ucx_inst_ptest/test_types | sort -R > $ucx_inst_ptest/test_types_short
+	cat $ucx_inst_ptest/test_types | grep -v cuda | sort -R > $ucx_inst_ptest/test_types_short
+	cat $ucx_inst_ptest/test_types | grep cuda | sort -R > $ucx_inst_ptest/test_types_short_cuda
 
 	UCX_PERFTEST="$ucx_inst/bin/ucx_perftest \
 					-b $ucx_inst_ptest/test_types_short \
 					-b $ucx_inst_ptest/msg_pow2_short -w 1"
 
+	UCX_PERFTEST_CUDA="$ucx_inst/bin/ucx_perftest \
+					-b $ucx_inst_ptest/test_types_short_cuda \
+					-b $ucx_inst_ptest/msg_pow2_short -w 1 -m cuda"
```

Comment:
can we add `-m cuda` to the batch file, instead of another command line?
we really need to test only tag matching with cuda. no reason to run all ucp/uct tests